src/doom/g_game.c

```diff
@@ -2672,6 +2685,13 @@ G_InitNew
                 skytexturename = "SKY3";
             }
             break;
+          case 6:        // [crispy] Sigil II
+            skytexturename = "SKY6_ZD";
+            if (R_CheckTextureNumForName(DEH_String(skytexturename)) == -1)
+            {
+                skytexturename = "SKY3";
+            }
```

Comment:
This is only needed for the first Sigil, which uses `SKY3` in `SIGIL_COMPAT_*.wad` instead of `SKY5` in `SIGIL_*.wad`.

```diff
diff --git a/src/doom/g_game.c b/src/doom/g_game.c
index e4c9479b..b1cdbdc5 100644
--- a/src/doom/g_game.c
+++ b/src/doom/g_game.c
@@ -2687,10 +2687,6 @@ G_InitNew
             break;
           case 6:        // [crispy] Sigil II
             skytexturename = "SKY6_ZD";
-            if (R_CheckTextureNumForName(DEH_String(skytexturename)) == -1)
-            {
-                skytexturename = "SKY3";
-            }
             break;
         }
         skytexturename = DEH_String(skytexturename);
```

Reply:
I was thinking of this more as a fallback in the event `SKY6_ZD` is missing. (Maybe in a future episode 6 replacement?)

Reply:
Oh yeah, that makes sense to me.